src/main/java/com/mraof/minestuck/entity/underling/UnderlingEntity.java

```diff
@@ -74,8 +74,7 @@ protected void registerGoals()
 		goalSelector.addGoal(5, new RandomWalkingGoal(this, 0.6D));
 		goalSelector.addGoal(6, new LookAtGoal(this, PlayerEntity.class, 8.0F));
 		goalSelector.addGoal(7, new LookRandomlyGoal(this));
-		
-		targetSelector.addGoal(1, new HurtByTargetAlliedGoal(this, entity -> MSTags.EntityTypes.UNDERLINGS.contains(entity.getType())));
```

Comment:
I'd assume that it's some sort of improvement to replace this with `HurtByTargetGoal` in each individual underling, but do you have some more background on this change and what it's for?